packages/@expo/metro-config/src/transform-worker/asset-transformer.ts

```diff
@@ -53,6 +53,7 @@ export async function transform(
 
   // Is bundling for webview.
   const isDomComponent = options.platform === 'web' && options.customTransformOptions?.dom;
+  const useMd5Filename = options.customTransformOptions?.useMd5Filename;
```

Comment:
Just use `isExport` I guess, don't see any other codepaths where you need different export builds. Not entirely sure why this needs to change between prod/dev though.

Reply:
`isExport` also used for `export:embed` and we don't want to use md5 filename for export:embed output